lldb/test/Shell/SymbolFile/DWARF/x86/DW_AT_const_value.s

```diff
@@ -165,6 +172,11 @@
         .long   .Lulong_ptr-.Lcu_begin0 # DW_AT_type
         .uleb128 0xdeadbeefbaadf00d     # DW_AT_const_value
 
+        .byte   18                      # Abbrev DW_TAG_variable
+        .asciz  "data16"                # DW_AT_name
+        .long   .Lu128-.Lcu_begin0      # DW_AT_type
+        .asciz "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"     # DW_AT_const_value
```

Comment:
This doesn't sound right. Shouldn't this be a 128 bit integer value? I'd also recommend writing the variable in a way that's easily recognisable as "correct" in the test expectations. Since we don't have a way (AFAIK) to insert 128 bit asm constants, I'd probably do it by putting two `.quad 0xhex` statements here and then telling lldb to print the variable in hex (by adding an extra `target variable --format x data16` command)

Reply:
That's a good idea. I'll try that out!